roles/drupal9/molecule/default/converge.yml

```diff
@@ -9,6 +9,7 @@
     - systems_user: 'deploy'
     - force_settings: false
     - running_on_server: false
+    - php_webserver: 'apache2'
```

Comment:
Omit this for testing or change to nginx to match the role vars and also our overall strategy of "move away from apache and mysql when possible, use nginx and postgresql."